apps/docusaurus/docs/concepts/blockchain.md

```diff
@@ -68,7 +68,7 @@ The raw transaction includes the following fields:
 | Payload                                                                              | Indicates an action or set of actions Alice's behalf. In the case this is a Move function, it directly calls into Move bytecode on the chain. Alternatively, it may be Move bytecode peer-to-peer [move script](../reference/glossary.md#transaction-or-move-script). It also contains a list of inputs to the function or script. For this example, it is a function call to transfer an amount of Aptos Coins from Alice account to Bob's account, where Alice's account is implied by sending the transaction and Bob's account and the amount are specified as transaction inputs. |
 | [Gas unit price](../reference/glossary.md#gas-unit-price)                            | The amount the sender is willing to pay per unit of gas, to execute the transaction. This is represented as Octa or units of 10<sup>-8</sup> Aptos utility tokens.                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| [Maximum gas amount](../reference/glossary.md#maximum-gas-amount)                    | The maximum gas amount in Aptos utility tokens Alice is willing to pay for this transaction. Gas charges are equal to the base gas cost covered by computation and IO multiplied by the gas price. Gas costs also include storage with an Apt-fixed priced storage model. This is represented as Octa or units of 10<sup>-8</sup> Aptos utility tokens.                                                                                                                                                                                                                                 |
+| [Maximum gas amount](../reference/glossary.md#maximum-gas-amount)                    | The maximum gas amount in Aptos utility tokens Alice is willing to pay for this transaction. Gas charges are equal to the base gas cost covered by computation and IO multiplied by the gas price. Gas costs also include storage with an Apt-fixed priced storage model. This is represented as Octa or units of 10<sup>-8</sup> Aptos utility tokens.                                                                                                                                                                                                                                |
```

Comment:
Aptos utility token is mentioned

Reply:
Oh, this is docusaurus, so I'm not bothering.